src/Synology.Api.Client/Apis/DownloadStation/Task/DownloadStationTaskEndpoint.cs

```diff
@@ -43,12 +44,24 @@ public Task<DownloadStationTaskListResponse> ListAsync()
         {
             var queryParams = new Dictionary<string, string?>
             {
-                { "additional",  "detail,file" }
+                { "additional",  "detail,file,transfer" }
             };
 
             return _synologyHttpClient.GetAsync<DownloadStationTaskListResponse>(_apiInfo, "list", queryParams, _session);
         }
 
+        public async Task<DownloadStationTask> GetInfoAsync(string id)
```

Comment:
Do you know what happens when the `id` is not found? Does the request fail or does it return an empty list?

If it returns an empty list, I think it would be better to change the return type to nullable, e.g. `Task<DownloadStationTask?>` and then return `result.Tasks.FirstOrDefault();` so the consumer can handle it instead of throwing the exception: `InvalidOperationException: Sequence contains no elements`.

Reply:
The Api request send : `{"error":{"code":404},"success":false}`

And the current requester throw an Error.
```
Synology.Api.Client.Exceptions.SynologyApiException : 'The Synology API Request failed.
Error Code: "404"
Error Description: "Invalid task id"
API: "SYNO.DownloadStation.Task" 
Method: "getinfo" 
Version: "1"'
```

So that's why I don't return a nullable, and don't bother using FirstOrDefault